spark/v3.1/spark/src/main/java/com/netease/arctic/spark/ArcticSparkCatalog.java

```diff
@@ -172,7 +172,6 @@ public Table createTable(
         PrimaryKeySpec primaryKeySpec = PrimaryKeySpec.builderFor(icebergSchema)
             .addDescription(properties.get("primary.keys"))
             .build();
-        properties.remove("primary.keys");
```

Comment:
why  remove this line ?

Reply:
Because want to get  the primary key from the source table properties, if remove this primary key information, is difficult to get primary key from the source table